classloader-leak-prevention/src/test/java/se/jiderhamn/classloader/leak/prevention/MultiThreadedHttpConnectionManagerCleanUpTest.java

```diff
@@ -0,0 +1,49 @@
+package se.jiderhamn.classloader.leak.prevention;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+import com.sun.jersey.api.client.config.DefaultClientConfig;
+import com.sun.jersey.client.apache.ApacheHttpClient;
+
+import se.jiderhamn.classloader.leak.JUnitClassloaderRunner;
+import se.jiderhamn.classloader.leak.Leaks;
+import se.jiderhamn.classloader.leak.prevention.cleanup.MultiThreadedHttpConnectionManagerCleanUp;
+
+/**
+ * Test case for leaks caused by com.sun.jersey.client.apache.ApacheHttpClient failing to close org.apache.commons.httpclient.MultiThreadedHttpConnectionManager
+ * 
+ * @author Marian Petrik
+ */
+@RunWith(JUnitClassloaderRunner.class)
+public class MultiThreadedHttpConnectionManagerCleanUpTest extends PreventionsTestBase<MultiThreadedHttpConnectionManagerCleanUp>{
```

Comment:
Any reason this test class cannot be simplified to a subclass of `ClassLoaderPreMortemCleanUpTestBase` (see for example [DriverManagerCleanUpTest](https://github.com/mjiderhamn/classloader-leak-prevention/blob/master/classloader-leak-prevention/src/test/java/se/jiderhamn/classloader/leak/prevention/cleaup/DriverManagerCleanUpTest.java))?